### ISSUE TITLE
GSOC: Keptn K6 Integration

### ISSUE THREAD

**Issue by jainammm** [keptn-sandbox/k6-service#5]
<h1 align="center">Keptn K6 Integration</h1>

- [x] #1 
- [x] #3 
- [x] #4 
- [x] Setup GitHub Automation
- [x] #7 
- [x] #12 
- [x] #13
- [x] #14
- [x] Run K6 files with Simple K6 binary
- [x] Extend to K6 extensions

**Comment by agardnerIT** [comment]
The final 2 tasks are done right? This can be closed?

**Comment by jainammm** [comment]
yes... this issue was for tracking the whole project... will add the additional tasks with issue links in the description and will close it
